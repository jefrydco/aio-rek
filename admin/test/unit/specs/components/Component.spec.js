import { shallowMount, createLocalVue } from "@vue/test-utils";
import Vuetify from "vuetify";
import VeeValidate from "vee-validate";

import Component from "~/components/Component";

const localVue = createLocalVue();
localVue.use(Vuetify);
localVue.use(VeeValidate);

describe("Component", () => {
  it("renders to match snapshot", () => {
    const wrapper = shallowMount(Component, {
      localVue
    });
    expect(wrapper).toMatchSnapshot();
  });
});
