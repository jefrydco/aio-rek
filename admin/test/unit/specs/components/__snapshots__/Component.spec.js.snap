// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders to match snapshot 1`] = `
<div id="component" class="mb-3">
  <v-text-field-stub error-messages="" box="" label="Example" required="" clearable="" data-vv-name="example" data-vv-as="Example"></v-text-field-stub>
</div>
`;
